features/org.wso2.carbon.appmgt.store.feature/src/main/resources/admin-dashboard/site/themes/default/templates/businessowners-task/js/businessowners.js

```diff
@@ -275,15 +275,26 @@ $(document).on("click", ".owner-edit-button", function () {
 
 $(document).on("click", ".owner-delete-button", function () {
     var ownerId = $(this).data("ownerId");
+    var ownerName = $(this).closest('tr').find("td:first").text();
     $.ajax({
                url: context + '/apis/businessowners/' + ownerId,
                type: 'DELETE',
                contentType: 'application/json',
                dataType: 'json',
                success: function (response) {
-                   updateOwners();
-                   Showalert("Owner Deleted Successfully ", "alert-success", "statusSuccess");
-                   location.reload();
+                   if(response) {
+                       $(".alert").alert();
+                       var isConfirmed = confirm("Are you sure you want to delete the business owner " + ownerName + "?");
+                       if (isConfirmed) {
+                           updateOwners();
+                           Showalert("Business Owner : " + ownerName + " Deleted Successfully ", "alert-success", "statusSuccess");
```

Comment:
This alert msg is not consistent. Aa a best practice, we don't start intermediate words with "Caps" with log or alert msges as they are shown to the user.